doc/manual/source/user_guide/building_enzo.rst

```diff
@@ -22,7 +22,7 @@ compiler, the following libraries are necessary:
    * `MPI <http://www.mcs.anl.gov/research/projects/mpi/>`_, for multi-processor parallel
      jobs.  Note that Enzo will compile without MPI, but it's fine to compile
      with MPI and only run on a single processor.
-   * `hg <https://www.mercurial-scm.org/>`_, a free, distributed
+   * `git <https://git-scm.org/>`_, a free, distributed
```

Comment:
The docs build is failing because there are multiple links with `git` as the target text. Adding the second underscore will fix that.
```suggestion
   * `git <https://git-scm.org/>`__, a free, distributed
```